common/views/components/PaletteColorPicker/PaletteColorPicker.tsx

```diff
@@ -185,7 +185,8 @@ const PaletteColorPicker: FunctionComponent<Props> = ({
               : ''
           }
         />
-      ) : (
+      )}
+      {isEnhanced && (
```

Comment:
Because we use this method elsewhere and I just want to make sure; was this causing an bug?

Reply:
it was causing the repeated gallery bug. Honestly, I'm starting to understand why we didn't allow filtering by colour for the no-Js experience